### DISCUSSION TITLE
Projects Beta - Persist title in overlay view when scrolling down

### DISCUSSION THREAD

**Discussion by conorriches** [community/community#38557] [Projects and Issues] [answered]
## Current behaviour:
When viewing an issue in the overlay view, and scrolling down, the title of the issue disappears, meaning the key context of the issue is no longer visible:

![githubissue](https://user-images.githubusercontent.com/905676/200556527-e1887402-5a78-4bac-875e-bf9b4d95215a.png)

## Expected behaviour:
When scrolling on an issue in the overlay view, persist the header (or just title), with a fixed position so that key context about the issue isn't lost.

#### Persisting the header:
![image](https://user-images.githubusercontent.com/905676/200557133-b027f48f-4d65-4307-966c-c193c141f507.png)

#### Persisting just the title:
![image](https://user-images.githubusercontent.com/905676/200558902-c419053c-d9d6-4fcf-b11e-2422cdb8e571.png)

## Why
So that when reviewing tickets either for backlog review or sprint planning, the title remains visible so key context isn't lost.

**Comment by mattcosta7** [comment, marked as answer]
This should be updated now to stay sticky while scrolling

**Reply by conorriches** [reply]
Many thanks! :)
